### ISSUE TITLE
SelectorFormat refuses valid BEM

### ISSUE THREAD

**Issue by mrpavlikov** [brigade/scss-lint#254]
New SelectorFormat linter refuses valid BEM selectors (modifications on elements):

``` css
.b-block__element-name_modifier_value {
    border: 1px solid;
}

.b-block__element-name_modifier {
    border: 1px solid;
}
```

Saying:

```
Selector `b-block__element-name_modifier_value` should be written in BEM (Block Element Modifier) format
Selector `b-block__element-name_modifier` should be written in BEM (Block Element Modifier) format
```

Am forced to use custom regular expression for this, would prefer to use built-in feature.

``` regexp
^[a-z]([-]?[a-z0-9]+)*(__[a-z0-9]([-]?[a-z0-9]+)*)?(_[a-z0-9]+){0,2}$
```
